srcpkgs/python3-ansible-lint/template

```diff
@@ -1,18 +1,18 @@
 # Template file for 'python3-ansible-lint'
 pkgname=python3-ansible-lint
-version=5.4.0
+version=6.0.0a1
 revision=1
 wrksrc="${pkgname/python3-/}-${version}"
 build_style=python3-pep517
 make_install_target="ansible_lint-${version}-py3-none-any.whl"
 hostmakedepends="python3-wheel python3-toml"
-depends="ansible-core python3-yamllint python3-six python3-yaml python3-ruamel.yaml python3-tenacity python3-enrich python3-wcmatch"
+depends="ansible-core python3-yamllint python3-six python3-yaml python3-ruamel.yaml python3-tenacity python3-enrich python3-wcmatch python3-ansible-compat"
 short_desc="Linter for Ansible files"
 maintainer="Jan Christian Grünhage <jan.christian@gruenhage.xyz>"
-license="MIT"
+license="GPL-3.0-only"
```

Comment:
> The ansible-lint project is distributed as [GPLv3](https://github.com/ansible-community/ansible-lint/blob/main/COPYING) due to use of [GPLv3](https://github.com/ansible-community/ansible-lint/blob/main/COPYING) runtime dependencies, like ansible and yamllint.
> 
> For historical reasons, its own code-base remains licensed under a more liberal [MIT](https://github.com/ansible-community/ansible-lint/blob/main/docs/licenses/LICENSE.mit.txt) license and any contributions made are accepted as being made under original [MIT](https://github.com/ansible-community/ansible-lint/blob/main/docs/licenses/LICENSE.mit.txt) license.

Reply:
Could you leave this as a comment in the template, so if in the future someone is confused about this, they don't have to dig through all PRs?

Reply:
done